src/core/Callout/index.tsx

```diff
@@ -59,29 +66,66 @@ const Callout = ({
     }
   };
 
+  const getAction = () => {
+    if (expandable) {
+      return (
+        <ButtonIcon
+          onClick={() => {
+            setStage(expanded ? SDS_STAGE_CLOSED : SDS_STAGE_OPEN);
+          }}
+          sdsSize="small"
+          sdsType="tertiary"
+        >
+          <Icon
+            sdsIcon={expanded ? "chevronUp" : "chevronDown"}
+            sdsSize="s"
+            sdsType="button"
+          />
+        </ButtonIcon>
+      );
+    }
+    return onClose ? (
+      <ButtonIcon
+        onClick={handleClose}
+        sdsSize="small"
+        sdsType="tertiary"
+        size="large"
+      >
+        {" "}
+        <Icon sdsIcon="xMark" sdsSize="s" sdsType="iconButton" />{" "}
+      </ButtonIcon>
+    ) : null;
+  };
+
+  const expanded = !expandable || stage === "open";
+
+  let calloutTitle = children;
+  let calloutContent;
+
+  if (Array.isArray(children)) {
+    if (children[0].type.name === "CalloutTitle") {
```

Comment:
would you need fallbacks here in case children is empty? for example `children[0]?.type.name`. not sure if that's a state you can get in.

Reply:
yea I think that makes sense.  I'm not sure how likely that state is either, but it seems like it doesn't hurt.  I'll also add some notes to the PR description on how these conditions work in terms  of what is shown based on what the user provides.

Reply:
yea let me fix this up a bit.  I think I actually don't need either of the `calloutTitle = children`, because that's already assigned above.  I can just move the explanatory comments up to that variable initialization.